.github/workflows/release.yml

```
@@ -11,40 +22,134 @@ permissions:
           GOOS=darwin GOARCH=amd64 go build -ldflags="-s -w" -o dist/up-darwin-amd64 ./cmd/up
           lipo -create -output dist/up-darwin-universal dist/up-darwin-arm64 dist/up-darwin-amd64
 
-      # Integration tests skipped in CI (requires keychain setup)
-      # Run locally with: ./integration-tests.sh
+          # Checksums
+          (cd dist && shasum -a 256 paw-proxy-darwin-arm64 paw-proxy-darwin-amd64 paw-proxy-darwin-universal up-darwin-arm64 up-darwin-amd64 up-darwin-universal > checksums.txt)
 
-      - name: Create Release
+      - name: Create release
+        if: steps.version.outputs.skip != 'true'
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)